build.gradle

```diff
@@ -91,6 +91,9 @@ apply plugin: 'org.spongepowered.event-impl-gen'
 genEventImpl {
     outputFactory = 'org.spongepowered.api.event.SpongeEventFactory'
 
+    inclusiveAnnotation 'org.spongepowered.api.util.annotation.GenerateFactoryMethod'
+    exclusiveAnnotation 'org.spongepowered.api.util.annotation.AbstractEvent'
```

Comment:
Could you rename this to something other than `AbstractEvent`? We already have an `AbstractEvent` class that serves as an abstract base class for event impls.

Reply:
So `NoFactoryMethod`?

Reply:
Sounds good to me